templates/quickstart-database-for-atlassian-services.yaml

```diff
@@ -160,19 +165,28 @@ Resources:
         DBName: ''
         DBPort: '5432'
         EnableEventSubscription: 'false'
-        Subnet1ID: !Select [0, !Split [ ",", !ImportValue ATL-PriNets]]
-        Subnet2ID: !Select [1, !Split [ ",", !ImportValue ATL-PriNets]]
+        Subnet1ID: !Select
```

Comment:
Using Exported values. I had to change the structure because it is not possible to use `!ImportValue !Sub`, it needs to be written in `Fn::ImportValue: !Sub` form. And because the whole statement is wrapped in a function, it needs to be broken into a list on multiple lines.